microsoft-edge/extensions-chromium/developer-guide/native-messaging.md

```diff
@@ -97,20 +128,24 @@ Sideload your extension to test native messaging with the host.  To sideload you
 
 1. Copy the key from `microsoft_catalog_extension_id` (ID) from the extension listing on the page.
 
-When you're ready to distribute your extension to users, publish your extension to the Microsoft Edge Add-ons website.  The extension ID of the published extension might differ from the ID that's used while sideloading your extension.  If the ID changed, update `allowed_origins` in the host manifest file with the ID of your published extension.
+When you're ready to distribute your extension to users, publish your extension to the Microsoft Edge Add-ons website.  The extension ID of the published extension might differ from the ID that's used while sideloading your extension.  If the ID changed, update `allowed_origins` in the native messaging host manifest file with the ID of your published extension.
 
 
 <!-- ====================================================================== -->
 ## Step 3 - Copy the native messaging host manifest file to your system
 
-The final step involves copying the native messaging host manifest file to your computer, and ensuring the manifest file is correctly configured.  To ensure your manifest file is placed in the expected location, do the following steps.  The location varies by platform.
+The final step involves copying the native messaging host manifest file to your computer, and making sure that this manifest file is correctly configured.  To ensure your native messaging host manifest file is placed in the expected location, do the following steps.  The location varies by platform.
 
 > [!NOTE]
-> Make sure that you provide read permissions on the manifest file, and run permissions on the host runtime.
+> Make sure that:
```

Comment:
These 2 points are applicable to only Mac and Linux. So, we should write the following:
On Linux and Mac, make sure that: ...

Reply:
rewrote/reformatted